sdk/cosmos/azure_data_cosmos/src/models/item.rs

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+use azure_core::Model;
+use serde::de::DeserializeOwned;
+
+/// Returned by Cosmos DB APIs that return a single item.
+///
+/// In some circumstances, an API that _can_ return an item will **not** return an item.
+/// For example, you can use [`ItemOptions`](crate::options::ItemOptions) to configure APIs
+/// that write new or updated items to avoid returning the updated item.
+// Marking this non-exhaustive would make things very difficult for users.
```

Comment:
Nit: will the compiler complain if you put a new line here just to visually separate the public comment from internal comment? I did a double take initially. If not, no biggie. This is semantically correct.

Reply:
I typed it out with a newline, and `cargo fmt` removed it 😭 . Let me see if I can make it work.

Reply:
Naw, don't worry about it. Not a big deal.

Reply:
I didn't see a straightforward way to have rustfmt leave these alone, so I'll just leave it.